### PR TITLE
Added default snippets and their definition

### PR DESCRIPTION
**Pull request by eshanrnh**

### CODE REVIEW

File: Fundamentals/Design/Partial-View-Macro-Files/index.md [hunk 1]
```diff
@@ -46,7 +46,28 @@ To create a Partial View Macro File from the snippet, go to the **Settings** sec
 
 ![Created partial view from snippet](images/created-partial-view-macro-file-from-snippet.png)
 
-By default, the partial view is saved in the `Views/MacroPartials` folder.
+By default, the partial view is saved in the `Views/MacroPartials` folder. Umbraco provides the following snippets:
```
Comment (jonat123) [inline Fundamentals/Design/Partial-View-Macro-Files/index.md:49]: Would it make sense to say something like: "Umbraco provides the following Macro Partial snippets"?

What do you think? :)

File: Fundamentals/Design/Partial-Views/index.md [hunk 2]
```diff
@@ -37,7 +37,23 @@ To create a partial view from the snippet, go to the **Settings** section in the
 
 ![Created partial view from snippet](images/created-partial-view-from-snippet.png)
 
-By default, the partial view is saved in the `Views/Partials` folder in the solution.
+By default, the partial view is saved in the `Views/Partials` folder in the solution. Umbraco provides the following snippets:
```
Comment (jonat123) [inline Fundamentals/Design/Partial-Views/index.md:40]: And the same here as my comment above :-)

**Comment by jonat123** [comment]
@eshanrnh Looks good, i made two comments,
let me know what you think :D

File: Fundamentals/Design/Partial-View-Macro-Files/index.md (hunk 1 above)
Comment (eshanrnh) [reply Fundamentals/Design/Partial-View-Macro-Files/index.md:49]: Makes sense 😀

**Comment by eshanrnh** [comment]
> @eshanrnh Looks good, i made two comments, let me know what you think :D

Incorporated the comments. Merging in the PR 🙂

**Pull request merged**